creating/creating_index.adoc

```diff
@@ -235,12 +272,64 @@ however, is not very practical for sharing as the user will be able to only down
+
+Look at the mongodb example:
+
+```
+# docker images openshift/mongodb-24-centos7
+REPOSITORY                               TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
+docker.io/openshift/mongodb-24-centos7   latest              d7c0c18b0ae4        16 hours ago        593.3 MB
+```
+
+Without squashing, you can see complete history and how each of the layers occupies space.
+
+```
+# docker history docker.io/openshift/mongodb-24-centos7:latest
+IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
+d7c0c18b0ae4        About an hour ago   /bin/sh -c #(nop) CMD ["run-mongod"]            0 B
+63e2ba112add        About an hour ago   /bin/sh -c #(nop) ENTRYPOINT &{["container-en   0 B
+ca996db9c281        About an hour ago   /bin/sh -c #(nop) USER [184]                    0 B
+8593b9473058        About an hour ago   /bin/sh -c #(nop) VOLUME [/var/lib/mongodb/da   0 B
+5eca88b7872d        About an hour ago   /bin/sh -c touch /etc/mongod.conf && chown mo   0 B
+9439db8f40ad        About an hour ago   /bin/sh -c #(nop) ADD dir:f38635e83f0e6943cd3   17.29 kB
+12c60945cbac        About an hour ago   /bin/sh -c #(nop) ENV BASH_ENV=/usr/share/con   0 B
+e6073f9a949f        About an hour ago   /bin/sh -c #(nop) ENV CONTAINER_SCRIPTS_PATH=   0 B
+619bf2ae5ed8        About an hour ago   /bin/sh -c yum install -y centos-release-scl    342.6 MB
+ab5deeccfe21        About an hour ago   /bin/sh -c #(nop) EXPOSE 27017/tcp              0 B
+584ded9dcbca        About an hour ago   /bin/sh -c #(nop) LABEL io.k8s.description=Mo   0 B
+17e3bcd28e07        About an hour ago   /bin/sh -c #(nop) ENV MONGODB_VERSION=2.6 HOM   0 B
+807a1e9c5a7b        16 hours ago        /bin/sh -c #(nop) MAINTAINER SoftwareCollecti   0 B
+28e524afdd05        10 days ago         /bin/sh -c #(nop) CMD ["/bin/bash"]             0 B
+044c0f15c4d9        10 days ago         /bin/sh -c #(nop) LABEL name=CentOS Base Imag   0 B
+2ebc6e0c744d        10 days ago         /bin/sh -c #(nop) ADD file:6dd89087d4d418ca0c   196.7 MB
+fa5be2806d4c        7 months ago        /bin/sh -c #(nop) MAINTAINER The CentOS Proje   0 B
+
+```
+See how the history and size changes after squashing all layers in a single one (using the script link:#squash_tools[above]):
+
+```
+# docker history docker.io/openshift/mongodb-24-centos7:squashed
+IMAGE               CREATED             CREATED BY          SIZE                COMMENT
+90036ed9bd1d        58 minutes ago                          522.1 MB
+
+```
+
+* One of the biggest benefits of using layers is the posibility to reuse them. Images are usually squashed into a single big layer, which does not allow for pushing partial updates in individual layers; instead, the whole image needs to be pushed into the registry upon a change. The same is applies to pulling the image from the registry.
```

Comment:
It would be really cook if docker-squash could be enhanced to leave the from layer, so it would squash everything to the lowest layer, or allow the user to specify how many layers he wants to keep.

Then perhaps we could ship this as oci-squash.